src/components/main/Main.tsx

```diff
@@ -0,0 +1,127 @@
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../todoItem/TodoItem';
+import { TempTodo } from '../todoItem/TempTodo';
+import {
+  updateTodoComplited,
+  // updateTodoTitle,
+} from '../../api/todos';
+
+interface Props {
+  todos: Todo[] | null;
+  tempTodo: Todo | null;
+  showError: (errText: string | boolean) => void;
+  handleDeleteTodo: (id: number) => void;
+  loading: boolean;
+  loadingID: number;
+}
+export const Main: React.FC<Props> = ({
+  todos,
+  tempTodo,
+  showError,
+  handleDeleteTodo,
+  loading,
+  loadingID,
+}) => {
+  const handleUpdateTodoIsCompleted = async (
+    id: number,
+    complitedCurrVal: boolean,
+  ) => {
+    try {
+      await updateTodoComplited(id, {
+        completed: !complitedCurrVal,
+      // eslint-disable-next-line no-console
+      }).then(resp => console.log(resp));
+    } catch {
+      showError('update');
+    }
+  };
+
+  return (
+    <section className="todoapp__main">
+      {todos
+      && todos.map(({
+        title,
+        id,
+        completed,
+      }) => (
+        <TodoItem
+          loading={loading}
+          loadingID={loadingID}
+          key={id}
+          title={title}
+          id={id}
+          completed={completed}
+          onDelete={handleDeleteTodo}
+          onIsComplitedUpdate={handleUpdateTodoIsCompleted}
+        />
+      ))}
+      {tempTodo
+      && <TempTodo tempTodo={tempTodo} />}
+
+      {/* This is a completed todo */}
```

Comment:
Remove the commented out example code (lines 61-124) as it is not relevant to the implementation and may cause confusion.